2019-09-04 14:01:27,998 ERROR: 'Aluno' object is not iterable [in /home/cavalcantigor/workspace/python/flaskeleton-api/app/resources/__init__.py:39]
